.github/workflows/bot.yml

```diff
@@ -61,8 +61,9 @@ jobs:
     - uses: actions-rs/toolchain@v1
```

Comment:
need to change this one as well.

Reply:
I have a plan to change the holistic change for `bot.yml` in another pull request. I updated the summary to describe this background.
Planned change: https://github.com/awslabs/dynein/compare/main...StoneDot:dynein:improve-bot